package.json

```diff
@@ -59,7 +59,9 @@
     "@sentry/node": "^6.16.1",
     "@sentry/tracing": "^6.16.1",
     "a-nonce-generator": "^2.0.1",
+    "ansi-colors": "^4.1.1",
     "bcryptjs": "^2.4.3",
+    "cli-progress": "^3.10.0",
```

Comment:
do we need this?

Reply:
sebenarnya ga pake juga ga masalah sih mas, ini buat ngebantu aja udah berapa banyak data yg dah di update...